plugin-sysstat/lxqtsysstatutils.cpp

```diff
@@ -44,13 +44,14 @@ QString netSpeedToString(int value)
     return QStringLiteral("%1 %2B/s").arg(1 << (value % 10)).arg(prefix);
 }
 
-int netSpeedFromString(QString value)
+int netSpeedFromString(const QStringView& value)
```

Comment:
[Qt doc](https://doc.qt.io/qt-6/qstringview.html#details) says, "QStringViews should be passed by value, not by reference-to-const:". Would you please change this?

Reply:
Done

Reply:
Very good.